pom.xml

```diff
@@ -13,4 +13,55 @@
         <maven.compiler.target>8</maven.compiler.target>
     </properties>
 
+    <dependencies>
+        <dependency>
+            <groupId>org.testng</groupId>
+            <artifactId>testng</artifactId>
+            <version>7.7.1</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>io.rest-assured</groupId>
+            <artifactId>rest-assured</artifactId>
+            <version>5.3.0</version>
```

Comment:
Hi, Sofi! I would suggest to put the versions of all the dependencies to the <properties> tag, so that it would be easier to find and fix version if needed.